.travis.yml

```diff
@@ -4,8 +4,8 @@ sudo: required
 dist: trusty
 language: go
 go:
- - 1.8
  - 1.9
+ - "1.10"
```

Comment:
nit: Remove quote for code style consistency?

I'm fine with doing this in another PR.

Reply:
yaml can not parse `1.10` correctly without any quote. :)
FYI：https://twitter.com/davecheney/status/964669282855485440